TechnicalDocuments/FastlaneMatch.md

```diff
@@ -0,0 +1,121 @@
+Fastlane Match
+==============
+
+Fastlane Match is a tool for managing certificates and provisioning profiles centrally. It downloads certificates and profiles from a GitHub repository or Google Cloud Storage bucket. For safety centrally stored files are encrypted.
+
+Fastlane match [documentation](https://docs.fastlane.tools/actions/match/).
+
+## Several Targets and Types
```

Comment:
I guess it will make sense to add a section about general usage of `fastlane match`, like what you need to do to get certificates locally, essentially running it in readonly mode with correct parameters. And how certificates are stored right now (i.e. that appstore certificates are in separate branch by the name of appstore org)

Reply:
👍 

I will add a section about this tomorrow.